README.md

```diff
@@ -1,6 +1,6 @@
 # labeledpipe
 
-[![Build Status](<redacted>)](<redacted>)
+[![Build Status](https://travis-ci.org/factset/labeledpipe.svg?branch=master)](https://travis-ci.org/factset/labeledpipe)
 [![Coverage Status](<redacted>)](<redacted>)
```

Comment:
Yes, we get: [![codecov.io](https://codecov.io/github/factset/labeledpipe/coverage.svg?branch=master)](https://codecov.io/github/factset/labeledpipe?branch=master)

I was planning on adding it in a follow-up pull request, but given that codecov is part of our Travis CI setup, I would be fine adding it as part of the pull request. What would be your preference?